packages/platform/atoms/event-types/wrappers/EventAdvancedWebWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import { EventAdvancedTab } from "@calcom/features/eventtypes/components/tabs/advanced/EventAdvancedTab";
+import type { EventTypeSetupProps } from "@calcom/features/eventtypes/lib/types";
+import { trpc } from "@calcom/trpc/react";
+
+export type EventAdvancedWebWrapperProps = Pick<EventTypeSetupProps, "eventType" | "team">;
+
+const EventAdvancedWebWrapper = (props: EventAdvancedWebWrapperProps) => {
+  const connectedCalendarsQuery = trpc.viewer.connectedCalendars.useQuery();
+  const { data: user, isPending } = trpc.viewer.me.useQuery();
```

Comment:
will move further up the tree after https://github.com/calcom/cal.com/pull/16808#discussion_r1780564577 is merged

Reply:
https://github.com/calcom/cal.com/commit/e18da6c4a51ad3b7028dc6a4e5e95d94d9a6deff